standards/SPIFFE_Federation.md

```diff
@@ -307,6 +307,56 @@ As described in the [Trustworthiness of the Bundle Endpoint Server](#74-trustwor
 
+### 7.7 Processing claims asserted by foreign trust domains
+
+SPIFFE Federation allows workloads in one trust domain to authenticate SVIDs, and all the claims therein, issued by other trust domains. By an "authentic" claim, we mean specifically that it can be safely assumed that the foreign trust domain authority _asserted_ the claim. This is not the same thing as saying that the claim is _true_, or that it is _correctly interpreted_.
+
+Operators should use caution and good judgement in evaluating if and how they should process claims asserted in SVIDs from foreign trust domains. The recommended posture is not to process foreign claims that have not been evaluated accoring to the framework described here (i.e. default deny), then assert exceptions for specific claims from specific trust domains to be allowed if they pass muster.
+
+It bears repeating that implementations should never process any claims from SVIDs that fail authentication (except in a forensic context).
+
+#### 7.7.1 SVID Claims
+
+For the purpose of this discussion, a *claim* is an assertion about some attribute of the subject of an SVID, or about the SVID itself. The canonical SPIFFE claim is the SPIFFE ID, which is a claim about the subject of the SVID. An example of a claim about the SVID itself is the expiration time.
+
+SVID formats (to date including X.509 and JWT) afford flexibility to encode additional claims, including claims that have not been evaluated by any standards body.
+
+In considering how to process claims made by foreign trust domains, it is helpful to consider three categories of claims:
+
+  1. Claims required by and/or defined by the SPIFFE specifications for the SVID type
+  2. Claims that are standardized by the relevant standards body for the document format (i.e. ITU-T for X.509, IETF for JWT)
+  3. Non-standardized claims
+
+Claims in the first category are well-defined in their meaning, and in many cases implementations are required to process them to adhere to SPIFFE specifications.
+
+Claims in the second category have standardized meaning, but operators have some choice whether to and how to process them.
+
+Claims in the third category are not standarized, so operators need to consider them on a case-by-case basis, possibly including navigating different meanings by different trust domains. Needless to say, this category should be approached with the highest degree of caution.
+
+#### 7.7.2 Claim scope
+
+Some claims, like the SPIFFE ID are _explicitly_ scoped to a trust domain by their structure. Because SVIDs are always issued by a particular trust domain authority, _all_ the claims therein are _implicitly_ scoped to that trust domain. This scoping should be preserved when presenting claims to any decision-making process.
```

Comment:
```suggestion
Some claims, like the SPIFFE ID, are _explicitly_ scoped to a trust domain (i.e. the claim explicitly includes a trust domain). _All_ claims contained within an SVID, which do not contain an explicit scope, are assumed to be _implicitly_ scoped to the trust domain of the SPIFFE ID; this scoping should be preserved when presenting claims to any decision-making process.
```

Note that the issuer of an SVID need not be a member of the trust domain of the SVID, or any trust domain at all. In the original text, "that trust domain" appear to refer to the trust domain of the "trust domain authority", which may or may not be the same as the SPIFFE ID in the SVID, or may not be any trust domain.

Reply:
I guess here I was thinking that the signing authority is always an "authorized authority" for the trust domain of the SVID by virtue of the (trust domain, bundle) tuple, even if it is not itself a member of the trust domain.  But, that's probably a little less clear than the language you have.

Reply:
I do have some concerns about the idea that claims that contain an explicit scope are exempt from scoping to the trust domain.

For example, consider a claim that encodes a Kubernetes pod `<namespace>/<name>`.  I think the `<namespace>` part of this claim is arguably _an explicit scope_, but I still think the claim is implicitly scoped to the SPIFFE trust domain, and should not be exempt from having that scoping preserved when presenting the claim to a decision-making process. If anything, the scope is the intersection of the trust domain and any other explicit scope.